README.md

```diff
@@ -75,17 +75,27 @@ Documentation can be found at <https://scitools.org.uk/cf-units/docs/latest/>.
 ## Get in touch
 
 - Questions, ideas, general discussion or announcements
-  of related projects use the
-  [Google Group](https://groups.google.com/forum/#!forum/scitools-iris).
-- Report bugs, suggest features or view the source code on
-  [GitHub](https://github.com/SciTools/cf-units).
+  of related projects: use the
+  [Discussions space](https://github.com/SciTools/cf-units/discussions).
+- Report bugs:
+  [submit a GitHub issue](https://github.com/SciTools/cf-units/issues)
+- Suggest features: see our [contributing guide](.github/CONTRIBUTING.md)
```

Comment:
The first bullet point here has a full stop, but the others don’t.  They should probably be consistent.

Reply:
9db3ba7ccb91646797200b36c264d4a1d5138e84